docs/source/tutorials/intro.rst

```diff
@@ -0,0 +1,94 @@
+.. _Environment:
+
+Setting up the Environment
+==========================
+
+.. _VirtualMachine:
+
+Virtual Machine
+---------------
+
+We provide a virtual machine disk that is the base for the application that will
+be developed during this tutorial.
+
+You can download the disk using this `Link <https://drive.google.com/a/stanford.edu/file/d/1SE4_oDF2wg_7ltKOIdJuZIqkpHzLUkbm/view?usp=sharing>`_.
+
+Using the Downloaded Disk
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After downloading it, extract the ``.tar.gz`` file, create a new Virtual Machine at the virtualization client of your preference.
+
+The instructions below are for `Oracle VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_ .
+Oracle VirtualBox is available for Windows, OS X and Linux hosts.
+
+This file is not a complete Virtual Machine dump that can be imported but instead a disk.
+
+In order to use this disk, start by creating a new virtual Machine, select Type as ``Linux`` and Version as ``Ubuntu (64-bit)``.
+Configure the amount of memory to use (something greater or equal 2048MB should do it.
+Make sure to select ``Use an existing virtual hard disk file.`` and select the extracted ``.vmdk`` file.
+
+.. figure:: /_static/new_vm.png
+    :scale: 100 %
+    :align: center
+    :alt: Create new VM
```

Comment:
All links to `/_static/<img_file> ` should now go to `/_static/tutorials/<img_file>`

Reply:
Thanks for pointing this out, it should be resolved now